ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -104,6 +104,7 @@ export const AnacondaWizard = ({ onAddErrorNotification, toggleContextHelp, titl
             setStepNotification={setStepNotification}
             isInProgress={isInProgress}
             setIsInProgress={setIsInProgress}
+            isQuitReboot={conf["Installation System"].type == "BOOT_ISO"}
```

Comment:
Is this something that can be depended on? I'd imagine this is something we want a D-Bus call, not a conf prop...

So maybe add a TODO?

Reply:
In an internal issue number `INSTALLER-3394` @M4rtinK suggested using the config directly and config was already used for something else.

Reply:
I think that's the idea about the config file - it's always loaded & the values should not change, so we can avoid making a DBus API for everything.